source/Scripts/RootConfiguration.ps1

```diff
@@ -2,29 +2,31 @@ configuration RootConfiguration
 {
     #<importStatements>
 
+    $rsopCache = Get-DatumRsopCache
+
     $module = Get-Module -Name PSDesiredStateConfiguration
     & $module {
-        param (
-            [string]$ModuleVersion,
+        param(
+            [string]$BuildVersion,
             [string]$Environment
         )
-        $Script:PSTopConfigurationName = "MOF_$($Environment)_$($ModuleVersion)"
-    } $ModuleVersion, $environment
+        $Script:PSTopConfigurationName = "MOF_$($Environment)_$($BuildVersion)"
+    } $buildVersion, $environment
```

Comment:
I used $ModuleVersion because that's the variable that has the gitversion version. Why are you changing?